data/json/overmap/map_extras.json

```diff
@@ -99,7 +99,7 @@
     "type": "map_extra",
     "name": { "str": "Supply Drop" },
     "description": "Several supply crates were dropped here.",
-    "generator": { "generator_method": "map_extra_function", "generator_id": "mx_supplydrop" },
```

Comment:
I think you might also want to remove the C++ function if you are comfortable with it.

Reply:
i'd like to, but i won't be able to confirm i did it properly, as i can't build the game on my own